packages/breadboard-web/src/boards/text-generator.ts

```diff
@@ -4,94 +4,57 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Schema, base, board, code } from "@google-labs/breadboard";
-import { core } from "@google-labs/core-kit";
-
-const metadata = {
-  title: "Text Generator",
-  description:
-    "This is a text generator. It can generate text using various LLMs. Currently, it supports the following models: Google Gemini Pro and OpenAI GPT-3.5 Turbo.",
-  version: "0.0.2",
-};
-
-const inputSchema = {
-  type: "object",
-  properties: {
-    text: {
-      type: "string",
-      title: "Text",
-      description: "The text to generate",
-    },
-    useStreaming: {
-      type: "boolean",
-      title: "Stream",
-      description: "Whether to stream the output",
-      default: "false",
-    },
-    MODEL: {
-      type: "string",
+import {
+  board,
+  enumeration,
+  input,
+  output,
+} from "@breadboard-ai/build";
+import { invoke, code } from "@google-labs/core-kit";
+
+const text = input({
+  type: "string",
+  title: "Text",
+  description: "The text to generate",
+});
+
+const MODEL = input({
```

Comment:
Lowercase `model`, since this isn't really a constant in the sense we usually use CAPS for.